randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -540,16 +540,22 @@ Extra - room_id: [16]
       All of the following:
           Screw Attack
           Any of the following:
-              # Get up
               Space Jump
               All of the following:
-                  # Jump on frozen Space Pirates TODO: someone please get a video of this. Also, how much harder is this with Diffusion, worth splitting out from template?
-                  Can Freeze Enemies
+                  # Jump on frozen Space Pirates
+                  Can Freeze Enemies and Can Single Walljump
                   Any of the following:
-                      # HJ method. Some RNG involved due to pirates moving.
-                      High Jump and Stand On Frozen Enemies (Advanced)
-                      # Walljump method. Heavy RNG involved due to pirates jumping
-                      Stand On Frozen Enemies (Hypermode) and Wall Jump (Beginner)
+                      # Wall Jump req
+                      Wall Jump (Advanced)
+                      High Jump and Wall Jump (Intermediate)
+                  Any of the following:
+                      # Ice req
+                      All of the following:
+                          Stand On Frozen Enemies (Expert)
+                          # https://youtu.be/6ha7hZ0iW1I
```

Comment:
wrong video/WJ requirements! These were all supposed to be **normal** walljumps, not single wall jumps!

Reply:
not entirely sure what you are talking about here, This shaft is too wide to even do jump extends so you have to SWJ

I moved the SWJ template in with the other wall jump tricks for readability. but other than that I see no problems here.

Reply:
> so you have to SWJ

Nope
You can perfectly do this with HJ OR "beginner WJ" :)

Reply:
Either you are talking about the wrong room or you guys are doing something patcher side I don't know about, please elaborate